### ISSUE TITLE
Adjusting font size

### ISSUE THREAD

**Issue by vinsifroid** [olivierkes/manuskript#206]
It's just an idea i had while using manuskript (v0.5.0). I am someone who have difficulties to read small text on screen, i find more comfortable to have it bigger. It would be great if the font size could be adjustable, like be able to zoom in or zoom out the text. Take for example Firefox (but there are other applications using that) where it's enough to press `ctrl +` or `ctrl -` to make the general font size bigger or smaller.

Hope i was clear enough for you to understand :smile: 

PS: Your work on manuskript is awesome. Wish i know a bit of Python to help you out

**Comment by gedakc** [comment]
The font size can be changed in the menu **Edit -> Settings**.  Choose **Views** and the **Text editor** tab and then adjust the font for the **Editor** pane.

See also [Tutorial: Using Manusrkipt to Write a Fiction Novel - Settings](http://gedakc.users.sourceforge.net/display-doc.php?name=manuskript-novel-writing#s)

Use the **Fullscreen** entry on the left-hand-side to adjust the look and feel of text in fullscreen mode.

@vinsifroid does that answer your question?

**Comment by vinsifroid** [comment]
@gedakc  Yes thank you, that's what i wanted to do.
However i suggest that some hotkeys to be assigned to these commands (alias increase and decrease font size). It make it much comfortable than have to search through menus. Well that's my opinion.

**Comment by gedakc** [comment]
Okay, I'll leave this issue open as a request for two hotkeys to increase and decrease font size similar to Firefox.

**Comment by olivierkes** [comment]
Good idea. I'm thinking both `CTRL +` / `CTRL -` and `CTRL MouseWheelUp` / `CTRL MouseWheelDown`. Do you agree?

**Comment by gedakc** [comment]
Sounds good to me.

**Comment by vinsifroid** [comment]
It sounds great.

**Comment by gedakc** [comment]
I have confirmed that `CTRL MouseWheelUp / CTRL MouseWheelDown` works to increase/decrease the font size in Manuskript 0.7.0 released on Aug 15, 2018.

The key combination `CTRL + / CTRL -` currently does not affect the font size.
